reference/rebac_admin_backend.rst

```diff
@@ -0,0 +1,32 @@
+ReBAC Admin Backend
+=======================
+
+The following sections briefly describe the functionalities offered by the ReBAC Admin Backend.
```

Comment:
Please add a sentence or two to explain what ReBAC Admin API is and point to the authorisation docs. As a suggestion:

```
The ReBAC Admin API is a RESTFul API that provides various endpoints to query or 
manipulate relationships in JAAS ReBAC authorisation model.

. hint::
    For an explanation on Relationship-Based Access Control (ReBAC) check out :doc:`this <../explanation/authorisation>` explanation topic.

.. hint::
    As a reference on JAAS authorisation model, check out :doc:`this <../reference/authorisation_model>` reference topic.
```